### PR TITLE
refactor: update imports under assessments folder

### PR DESCRIPTION
**Pull request by smoralesd**
#### Description of changes

Update imports under assessments folder

#### Pull request checklist

- [x] Addresses an existing issue: Fixes #0000
  - does not exist
- [x] Added relevant unit test for your changes. (`yarn test`)
   - no code changes => no unit test updates
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
   - not affected
- [x] Ran precheckin (`yarn precheckin`)
- [x] (UI changes only) Added screenshots/GIFs to description above
   - not affected
- [x] (UI changes only) Verified usability with NVDA/JAWS
   - not affected

### CODE REVIEW

File: src/assessments/assessment-builder.tsx [hunk 1]
```diff
@@ -4,27 +4,27 @@ import * as _ from 'lodash';
...
     PropertyBags,
     VisualizationInstanceProcessor,
     VisualizationInstanceProcessorCallback,
-} from '../injected/visualization-instance-processor';
-import { DrawerProvider } from '../injected/visualization/drawer-provider';
-import { DictionaryStringTo } from '../types/common-types';
+} from 'injected/visualization-instance-processor';
+import { DrawerProvider } from 'injected/visualization/drawer-provider';
+import { DictionaryStringTo } from 'types/common-types';
+import { ManualTestStatus } from '../common/types/manual-test-status';
```
Comment (dbjorge) [inline src/assessments/assessment-builder.tsx]: missed one?

Comment (dbjorge) [inline src/assessments/assessment-builder.tsx]: It looks like you switched to the `common/...` style for most of the imports from common, but are still using `../` style for `manual-test-status` imports throughout the files in this PR - is this one meant to use `common/` as well?

File: src/assessments/automated-checks/build-test-steps-from-rules.tsx [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-import { NewTabLink } from '../../common/components/new-tab-link';
-import { Messages } from '../../common/messages';
 import { ManualTestStatus } from '../../common/types/manual-test-status';
-import { VisualizationType } from '../../common/types/visualization-type';
-import { AssessmentInstanceDetailsColumn } from '../../DetailsView/components/assessment-instance-details-column';
-import { AssessmentInstanceRowData } from '../../DetailsView/components/assessment-instance-table';
-import { RuleAnalyzerConfiguration } from '../../injected/analyzers/analyzer';
-import { AnalyzerProvider } from '../../injected/analyzers/analyzer-provider';
-import { DecoratedAxeNodeResult, ScannerUtils } from '../../injected/scanner-utils';
 import { ScannerRuleInfo } from '../../scanner/scanner-rule-info';
```
Comment (dbjorge) [inline src/assessments/automated-checks/build-test-steps-from-rules.tsx]: Should this switch to `../../`?

**Comment by dbjorge** [comment]
LGTM besides the 2 comments. Verified no other missed relative imports that traverse up to src/:

```
> git co update-imports
Already on 'update-imports'
Your branch is up to date with 'smoralesd/update-imports'.

> function dirsbetween($root, $file) { return $file.FullName.Substring($root.FullName.Length + 1).Split('\').Count - 1 }
> function findrootrelativeimports($root, $file) { $distance = dirsbetween $root $file; $uppattern = '\.\./'*$distance; Select-String -LiteralPath $file.FullName -Pattern "from '$uppattern"; }
> $src = Get-Item ..
> $fs = gci -File -Include '*.ts','*.tsx' -Recurse
> $fs | % { findrootrelativeimports $src $_ } | sort Path,LineNumber | ft Path,LineNumber,Line

Path                                                                                                            LineNumber  Line
----                                                                                                            ----------  ----
Q:\repos\accessibility-insights-web\src\assessments\assessment-builder.tsx                                              27  import { ManualTestStatus } from '../common/types/manual-test-status';
Q:\repos\accessibility-insights-web\src\assessments\assessment-default-message-generator.tsx                             7  import { ManualTestStatus } from '../common/types/manual-test-status';
Q:\repos\accessibility-insights-web\src\assessments\automated-checks\automated-checks-visualization-enabled-toggle.tsx   7  import { ManualTestStatus } from '../../common/types/manual-test-status';
Q:\repos\accessibility-insights-web\src\assessments\automated-checks\build-test-steps-from-rules.tsx                    13  import { ManualTestStatus } from '../../common/types/manual-test-status';
Q:\repos\accessibility-insights-web\src\assessments\automated-checks\build-test-steps-from-rules.tsx                    14  import { ScannerRuleInfo } from '../../scanner/scanner-rule-info';
Q:\repos\accessibility-insights-web\src\assessments\types\requirement.ts                                                18  import { ManualTestStatus } from '../../common/types/manual-test-status';
```

File: src/assessments/assessment-builder.tsx (hunk 1 above)
Comment (smoralesd) [reply src/assessments/assessment-builder.tsx]: Good catch! Updated.

File: src/assessments/automated-checks/build-test-steps-from-rules.tsx (hunk 2 above)
Comment (smoralesd) [reply src/assessments/automated-checks/build-test-steps-from-rules.tsx]: Good catch! Updated.

**Pull request merged**